data/resume/resume.json

```diff
@@ -16,7 +16,7 @@
       "startDate": "2021-08-01",
       "highlights": [
         "Built functional React components (utilizing hooks when appropriate), elevated common state and async handling to redux reducers, and extracted components as sensible patterns of usage emerged.",
-        "Utilize Grubhub’s component library for common cases to solve design needs, maintain consistency of user experience, and speed up development. For situational needs, have leveraged SCSS or Styled Components.",
```

Comment:
I think the apostrophe character used here was a copy pasta from word, updated to use normal character